libraries/freertos_plus/aws/greengrass/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ afr_module_dependencies(
     PUBLIC
         AFR::secure_sockets
     PRIVATE
-        AFR::mqtt
+        AFR::core_mqtt
```

Comment:
We didn't update this when updating GG Demo? How does that work?

Reply:
This file isn't under the demos directory, it's under libraries, so it wasn't updated along with the demo

Reply:
Was `AFR::mqtt` earlier bringing dependency on `AFR::core_mqtt` indirectly?

Reply:
Yes, I just replaced it in case MQTT was still needed

Reply:
I did a quick scan of the includes in the Greengrass files, and I don't see any MQTT dependency. If MQTT is not required for the library, it would be better to remove it from the dependencies of this library as it otherwise, gives the wrong picture of the library

Reply:
Good point. I'll try that in a follow up PR when I remove the unused demo config files, as I don't want to make more changes like this to this PR after already testing everything